rcl_ext/README.md

```diff
@@ -0,0 +1,23 @@
+# The rcl_ext package
+
+The package rcl_ext is a small [ROS 2](http://www.ros2.org/) package for providing a thin API layer on top of RCL to create nodes, publishers, subscribers and timers with a one-liner.
+
+It it similar to rclc API, however does not provide a spin() funktion for the node to receive messages. This is intentional because the executor concept shall be implemented a different package, e.g. rcl-executor.
```

Comment:
Why not use rclc? (Besides the fact that the repository is currently not being updated).
It looks very similar. We could update rclc to use the new rcl_executor implementation.

In any case, I don't think that the package `rcl_ext` belongs in this repository. This repository is meant to contain the building blocks for full-featured ROS client libraries to use. I don't think an additional layer between `rcl` and `rclcpp` is necessary.

Reply:
Based on this and previous discussions, we will create a new pull request to rclc with the rcl_executor and rcl_ext. Will come up with better names, e.g. rcl_convenience  instead of rcl_ext.